doc/src/yaccextensions.md

```diff
@@ -3,8 +3,15 @@
 At the beginning of a `.y` file is a `%grmtools{}` section, by default this section is required.
 But a default can be set or forced by using a `YaccKindResolver`.
 
-## Required flags
-    * yacckind
+| Flag                      | Value     | Required     |
+|---------------------------|-----------|--------------|
+| `yacckind`                | YaccKind  | &checkmark;  |
+
+`YaccKind` Values
+* Grmtools
```

Comment:
Perhaps we should just link to the complete `YaccKind` descriptions in https://softdevteam.github.io/grmtools/master/book/yacccompatibility.html#yacckinds -- or maybe move them here or ... ?

Reply:
I think the main reason I was including it directly here, is because of the parsing code,
It doesn't take the `YaccKind::` or the `YaccOriginalActionKind::` rust fully qualified module paths
and stuff (at least at the moment!).

In the same way that *nimbleparse* takes a `YaccKind` and converts it to a string.
So this documentation was kind of trying to be like the `nimbleparse` help text which also duplicates the names of the kinds.

I should probably make it also accept `YaccKind` qualified paths, and link to that?

Reply:
Works for me!

Reply:
Do we need to add test(s) for the code in be067d9?

Reply:
Probably a good idea.

Reply:
Added in [91794aa](https://github.com/softdevteam/grmtools/pull/499/commits/91794aa99e7c94c7d0558dfc0b474f1272df7d91)